akd_local_auditor/src/auditor/mod.rs

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+//! This module holds the auditor operations based on binary-encoded AuditProof blobs
+
+use anyhow::{anyhow, bail, Result};
+use clap::{Parser, Subcommand};
+use log::{debug, error, info, warn};
+use std::marker::{Send, Sync};
+
+use rustyline::error::ReadlineError;
+use rustyline::Editor;
+
+const HISTORY_FILE: &str = ".akd_local_auditor_history";
+
+fn format_qr_record<H>(p_hash: H::Digest, c_hash: H::Digest, epoch: u64) -> Vec<u8>
+where
+    H: winter_crypto::Hasher,
+{
+    let p_bytes = akd::serialization::from_digest::<H>(p_hash);
+    let c_bytes = akd::serialization::from_digest::<H>(c_hash);
+    let epoch_bytes = epoch.to_ne_bytes();
```

Comment:
What's the reason for native endianness here?

Reply:
ah total typo, that should be le bytes. my bad!